tests/SprykerEcoTest/Zed/Easycredit/_support/EasycreditTester.php

```diff
@@ -0,0 +1,147 @@
+use Orm\Zed\Sales\Persistence\Map\SpySalesOrderTableMap;
+use Orm\Zed\Sales\Persistence\SpySalesOrder;
+use Orm\Zed\Sales\Persistence\SpySalesOrderAddress;
+use Orm\Zed\Sales\Persistence\SpySalesOrderTotals;
+
+/**
+ * Inherited Methods
+ *
+ * @method void wantToTest($text)
+ * @method void wantTo($text)
+ * @method void execute($callable)
+ * @method void expectTo($prediction)
+ * @method void expect($prediction)
+ * @method void amGoingTo($argumentation)
+ * @method void am($role)
+ * @method void lookForwardTo($achieveValue)
+ * @method void comment($description)
+ * @method void pause()
+ *
+ * @SuppressWarnings(PHPMD)
+ */
+class EasycreditTester extends Actor
+{
+    use _generated\EasycreditTesterActions;
+
+    /**
+     * @return int
+     */
+    public function createOrder(): int
+    {
+        $salesOrderEntity = new SpySalesOrder();
+
+        $this->addOrderDetails($salesOrderEntity);
+        $this->addAddresses($salesOrderEntity);
+        $salesOrderEntity->save();
+
+        $this->addOrderTotals($salesOrderEntity);
+
+        return $salesOrderEntity->getIdSalesOrder();
+    }
+
+    /**
+     * @param \Orm\Zed\Sales\Persistence\SpySalesOrder $salesOrderEntity
+     *
+     * @return void
+     */
+    protected function addOrderDetails(SpySalesOrder $salesOrderEntity): void
+    {
+        $salesOrderEntity->setOrderReference(random_int(0, 9999999));
+        $salesOrderEntity->setCurrencyIsoCode('EUR');
+        $salesOrderEntity->setPriceMode(null);
+        $salesOrderEntity->setIsTest(true);
+        $salesOrderEntity->setSalutation(SpySalesOrderTableMap::COL_SALUTATION_MR);
+        $salesOrderEntity->setFirstName('FirstName');
+        $salesOrderEntity->setLastName('LastName');
+    }
+
+    /**
+     * @param \Orm\Zed\Sales\Persistence\SpySalesOrder $salesOrderEntity
+     *
+     * @return void
+     */
+    protected function addAddresses(SpySalesOrder $salesOrderEntity): void
+    {
+        $billingAddressEntity = $salesOrderEntity->getBillingAddress();
+        if ($billingAddressEntity === null) {
+            $billingAddressEntity = $this->createBillingAddress();
+            $salesOrderEntity->setBillingAddress($billingAddressEntity);
+        }
+
+        $shippingAddressEntity = $salesOrderEntity->getShippingAddress();
+        if ($shippingAddressEntity === null) {
+            $salesOrderEntity->setShippingAddress($billingAddressEntity);
+        }
+    }
+
+    /**
+     * @return \Orm\Zed\Country\Persistence\SpyCountry
+     */
+    protected function getCountryEntity(): SpyCountry
+    {
+        $countryQuery = new SpyCountryQuery();
+        $countryQuery->filterByIso2Code('DE');
+        $countryQuery->filterByIso3Code('DEU');
+        $countryQuery->filterByName('Germany');
+        $countryQuery->filterByPostalCodeMandatory(true);
+        $countryQuery->filterByPostalCodeRegex('\d{5}');
+
+        $countryEntity = $countryQuery->findOneOrCreate();
+        $countryEntity->save();
```

Comment:
Would it make sense to remove changes from DB after test is executed?
E.g.
```
$this->getDataCleanupHelper()->_addCleanup(function () use ($countryEntity): void {
    SpyCountryQuery::create()
            ->filterByIdCountry($countryEntity->getIdCountry())
            ->delete();
});
```

The same goes for similar cases.